Source/items.cpp

```diff
@@ -1295,10 +1295,8 @@ void GetStaffSpell(const Player &player, Item &item, int lvl, bool onlygood)
 			s = static_cast<int8_t>(SpellID::Firebolt);
 	}
 
-	int minc = GetSpellData(bs).sStaffMin;
-	int maxc = GetSpellData(bs).sStaffMax - minc + 1;
 	item._iSpell = bs;
-	item._iCharges = minc + GenerateRnd(maxc);
+	item._iCharges = RndBetween(GetSpellData(bs).sStaffMin, GetSpellData(bs).sStaffMax);

```

Comment:
You can also use this function as a drop-in replacement for `RndPl()` in this file.

Reply:
I know, didn't want to cause unnecessary diff :)